web/waitlist/styles.css

```
@@ -0,0 +1,333 @@
+  margin: 18px 0 0;
+  padding: 0;
+  border: 0;
+  gap: 22px;
+}
+
+legend {
+  position: absolute;
+  width: 1px;
+  height: 1px;
+  overflow: hidden;
+  clip: rect(0 0 0 0);
```

Stylelint (17.14.0)

[error] Deprecated property "clip" (property-no-deprecated)

(property-no-deprecated)